### DISCUSSION TITLE
Ecommerce Implementation - Need Ideas

### DISCUSSION THREAD

**Discussion by HBSTech** [HBSTech/Kentico13CoreBaseline#2] [General]
What are you all looking for in terms of features for a Generic, easy to modify, ecommerce baseline solution.

Our goal is to make it where all the pieces of the ecommerce are there and ready for you to customize.  Post your ideas here!

**Comment by rafik4000** [comment]
Hi trivor, i believe what is missing is products page, the cart and check out cart is perfect but the products page that is missing.
